hbase-hbtop/README.md

```diff
@@ -0,0 +1,247 @@
+| #SF | Number of StoreFiles | true |
+| MEMSTORE | MemStore Size | true |
+| UHEAP | Used Heap Size | true |
+| MHEAP | Max Heap Size | true |
+
+### Changing mode
+
+You can change mode by pressing `m` key in the top screen.
+
+![Changing mode](img/changing_mode.gif "Changing mode")
+
+### Changing the refresh delay
+
+You can change the refresh by pressing `d` key in the top screen.
+
+![Changing the refresh delay](img/changing_refresh_delay.gif "Changing the refresh delay")
+
+### Changing the displayed fields
+
+You can move to the field screen by pressing `f` key in the top screen. In the fields screen, you can change the displayed fields by choosing a field and pressing `d` key or `space` key.
+
+![Changing the displayed fields](img/changing_displayed_fields.gif "Changing the displayed fields")
+
+### Changing the sort field
+
+You can move to the fields screen by pressing `f` key in the top screen. In the field screen, you can change the sort field by choosing a field and pressing `s`. Also, you can change the sort order (ascending or descending) by pressing `R` key.
+
+![Changing the sort field](img/changing_sort_field.gif "Changing the sort field")
+
+
+### Changing the order of the fields
+
+You can move to the fields screen by pressing `f` key in the top screen. In the field screen, you can change the order of the fields.
+
+![Changing the order of the fields](img/changing_order_of_fields.gif "Changing the sort field")
+
+### Filters
+
+You can filter the metric records with the filter feature. We can add filters by pressing `o` key for ignoring case or `O` key for case sensitive.
+
+![Adding filters](img/adding_filters.gif "Adding filters")
+
+The syntax is as follows:
+```
+<Field><Operator><Value>
+```
+
+For example, we can add filters like the following:
+```
+NAMESPACE==default
+REQ/S>1000
+```
+
+The operators we can specify are as follows:
+
+| Operator | Description |
+|---|---|
+| = | Partial match |
+| == | Exact match |
+| > | Greater than |
+| >= | Greater than or equal to |
+| < | Less than |
+| <= | Less than and equal to |
+
+You can see the current filters by pressing `^o` key and clear them by pressing `=` key.
+
+![Showing and clearing filters](img/showing_and_clearing_filters.gif "Showing and clearing filters")
+
+### Drilling down
+
+You can drill down the metric record by choosing a metric record that you want to drill down and pressing `i` key in the top screen. With this feature, you can find hot regions easily in a top-down manner.
+
+![Drilling down](img/driling_down.gif "Drilling down")
+
+### Help screen
+
+You can see the help screen by pressing `h` key in the top screen.
+
+![Help screen](img/help_screen.gif "Help screen")
+
+## Others
+
+### How hbtop gets the metrics data
+
+hbtop gets the metrics from ClusterMetrics from Admin#getClusterMetrics()  because that needs to access only HBase Master to do that. So, when we add more metrics to hbtop, we first need to add more metrics to ClusterMetrics.
```

Comment:
"hbtop gets the metrics from ClusterMetrics from Admin#getClusterMetrics()  because that needs to access only HBase Master to do that. So, when we add more metrics to hbtop, we first need to add more metrics to ClusterMetrics."

Suggest the below in stead (dropping the scaling concern from the README... that can go into the JIRA).

"hbtop gets the metrics from ClusterMetrics which is returned as the result of a call to Admin#getClusterMetrics() on the current HMaster. To add metrics to hbtop, they will need to be exposed via ClusterMetrics."

Reply:
Thanks. Modified.